assets/src/story-embed-block/block/save.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import StoryPlayer from './storyPlayer';
+
+function save({ attributes }) {
+  const { url, title, poster, width, height, align = 'none' } = attributes;
+
+  if (!url || !title) {
+    return null;
+  }
+
+  return (
+    <div className={`wp-block-web-stories-embed align${align}`}>
```

Comment:
The original plan was to create a dynamic block and not save content directly into the post, curious why the change? I guess currently the script and style are always loaded now, even if the block is not used in a post?

Reply:
> The original plan was to create a dynamic block and not save content directly into the post, curious why the change?

I might change it again to a dynamic block, haven't decided yet. If we keep the output that simple, there wouldn't really be a need for a dynamic block.

> I guess currently the script and style are always loaded now, even if the block is not used in a post?

Yeah that's a limitation of how `register_block_type` registers blocks. I'd change that with a dynamic block.

Reply:
Made it a partially dynamic block now, keeping the `save()` function as a cached copy.